scriptsBMI.js

```diff
@@ -0,0 +1,93 @@
+document.getElementById("contentBMI").style.display = 'none';
+document.getElementById("rightContentBMI").style.display = 'none';
+
+function calculateResult() {
+    if (getWeight() == 0 || getHeight() == 0) {
+        const button = document.getElementById("calculate");
+        button.style.backgroundColor="#D6D6D6";
+    } else {
+        activateCalculateButton();
+        resultBMI();
+        displayResults(BMI, description);
```

Comment:
`displayResults(BMI, description);` jest wywoływane dwa razy.

najpierw w funkcji resultBMI(); a potem tutaj gdzie zaznaczyłem. Raz byłby wystarczający.

`resultBMI() `to kiepska nazwa funkcji. Powinien być w nazwie jakiś czasownik jeszcze, coś w stylu getBMIResult() lub calculateBMIResult(), lub cokolwiek innego opisującego co z tym rezultatem się dzieje